onnxruntime/server/request_id.cc

```diff
@@ -0,0 +1,20 @@
+#include "request_id.h"
+// boost random is using a deprecated header in 1.69
+// See: https://github.com/boostorg/random/issues/49
+#define BOOST_PENDING_INTEGER_LOG2_HPP
+#include <boost/integer/integer_log2.hpp>
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+namespace onnxruntime {
+namespace server {
+namespace util {
+std::string InternalRequestId() {
+  return boost::uuids::to_string(boost::uuids::random_generator()());
+}
+const std::string REQUEST_HEADER = "x-ms-request-id";
```

Comment:
I find it confusing that the relation to MS and ID is gone now. It makes it less readable in other source files. Why not be more explicit and name it MS_REQUEST_ID_HEADER?